editor/project_converter_3_to_4.cpp

```diff
@@ -398,6 +398,7 @@ static const char *gdscript_function_renames[][2] = {
 	{ "import_animation_from_other_importer", "_import_animation" }, //EditorSceneFormatImporter
 	{ "import_scene_from_other_importer", "_import_scene" }, //EditorSceneFormatImporter
 	{ "instance_set_surface_material", "instance_set_surface_override_material" }, // RenderingServer
+	{ "interpolate", "sample" }, // Curve, Curve2D, Curve3D, Gradient
```

Comment:
Aren't there other `interpolate` methods that will be wrongly changed?

Reply:
There's **GPUParticles2D/3D**.`interpolate()`. There's also **Variant**.`interpolate()` but I don't believe it is exposed anywhere...

Reply:
> There's GPUParticles2D/3D.interpolate()

This property only exists in 4.0, so it should be fine.